scripts/ci/qemu-run.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+if [ ! -f "./OVMF_CODE.fd" ]; then
+    wget -O ./OVMF_CODE.fd https://cloud.3mdeb.com/index.php/s/iXmyJxKLBB2Eb94/download
+fi
+
+if [ ! -f "./OVMF_VARS.fd" ]; then
+    wget -O ./OVMF_VARS.fd https://cloud.3mdeb.com/index.php/s/Mn2n92DZLnZQZ8N/download
+fi
```

Comment:
@macpijan I guess this is some release candidate, but in the future (with the v0.1.0 release), we plan to put that something going directly from dasharo.com.

Reply:
@pietrushnic Release v0.1.0 was apparently a couple of months ago, but no binaries wre published: https://docs.dasharo.com/variants/qemu_q35/releases/

Now that we have CI enabled, maybe I will just integrate one more GitHub action to publish github release on tag and link this instead?